src/backend/direct.rs

```diff
@@ -649,9 +650,9 @@ impl crate::Context for Context {
         wgc::gfx_select!(*device => global.device_create_bind_group(
             *device,
             &bm::BindGroupDescriptor {
-                label: desc.label,
+                label: desc.label.as_ref().map(|label| label[..].into()),
```

Comment:
these should be aligned with explicit `Cow::Borrowed` like the other ones

Reply:
I had hoped I wouldn't have to touch these again because they are in the implementation, but ok. 😄

Reply:
thank you!